ipp-v3-java-devkit/src/main/java/com/intuit/ipp/interceptors/PrepareRequestInterceptor.java

```diff
@@ -310,7 +310,7 @@ protected String getBaseUrl(String url) {
 		
 		if(context.getMinorVersion() == null)
 		{
-		context.setMinorVersion("47");
+		context.setMinorVersion("50");
```

Comment:
Let's not default to 50 as it is an unreleased version and in such cases it will default to 1 instead which we don't want.